doc/cabal-package-description-file.rst

```diff
@@ -1,6 +1,8 @@
-Package Description
-===================
+package.cabal File
```

Comment:
Does rst allow “`.cabal` File”?

I say for newcomers not to slip on the fact that `cabal.project` is a file with `.project` extension and `cabal` name, while `package.cabal` actually can have many names.

Reply:
Possibly `*.cabal file` here.

Reply:
How about a placeholder syntax like `<package>.cabal` to give a name to what will be there?

Reply:
Fine by me!

Reply:
I dislike titles starting from file names, commands and other technicalities like that, because I think they don't help to understand what the section is about — the main point of a title. At the same time, I agree that this particular title better make it clear that the section has to do with the cabal file. So, my preference would be sometime like:

```
Package description — <package>.cabal file
```
(or whatever the proper punctuation mark is in a view of a native speaker). 

I also (like David) prefer to explicitly placeholder syntax of some form. It's be nice, in the future, to make a pass over the whole manual and harmonize our approach in this regard.

Reply:
Sounds good. What about "Project description - cabal.project file" or "Multi-package description - cabal.project file" or "Multi-package project description - cabal.project file" to be consistent for the other file title?

Reply:
"Project description — cabal.project file" makes most sense to me. (Be sure to use m-dashes :pray:)

Reply:
Done